recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost >=1.56
+    - python
+    - setuptools >=18.0
+    - numpy x.x
```

Comment:
Unrelated question to the jinja templating, do you know if we can both restrict version and use `x.x`?

I didn't see a good example, but I did try a few things and didn't find any that really worked. Admittedly, I have seen this work on NumPy's as old as 1.7 (and that may not even be the lower limit) so it isn't a big deal if it can't be done. After all, it definitely will work with NumPy's we do build with.